src/main/java/org/killbill/billing/client/model/SubscriptionUsage.java

```diff
@@ -0,0 +1,76 @@
+package org.killbill.billing.client.model;
```

Comment:
> for the files we must add copyright header?

Yes, something similar to the other files to license them under the Apache License 2.0.

> RAT - what does it mean?

It's our auditing tool (http://creadur.apache.org/rat/), which makes sure all files are properly licensed.